### PR TITLE
Correct "in in" typos

### PR DESCRIPTION
**Pull request by Ironjanowar**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,11 +52,11 @@ You can contribute to Erlang/OTP by opening a Pull Request.
 
 Make sure you create a new branch for your pull request with `git checkout -b new-branch-name`.
 Give the branch a short but descriptive name, like `stdlib/lists-length-fix`.
-Never do your work directly on `maint` or `master`.
+Never do your work directly on `main` or `master`.
```
Comment (jhogberg) [inline CONTRIBUTING.md:55]: `maint` is on purpose. It's short for "maintenance."

File: lib/megaco/doc/standard/draft-ietf-megaco-h248v2-04.txt [hunk 2]
```diff
@@ -8903,7 +8903,7 @@
                                        Ref.: ITU-T Q.2931 
 
       SSCS        See      See table  See table in C.7 
-                  table in in C.7 
+                  table in C.7
```
Comment (jhogberg) [inline lib/megaco/doc/standard/draft-ietf-megaco-h248v2-04.txt:8906]: This is a standards document, we should not fix anything in it, be it typos or grammar.

**Review by jhogberg — CHANGES_REQUESTED** [review]
Thanks for the PR! Can you rebase this to `maint` so that the online documentation will be updated sooner? I've also left a few comments on changes that shouldn't be made.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Ironjanowar) [reply CONTRIBUTING.md]: Sorry! :sweat_smile:

**Comment by jhogberg** [comment]
Please `git rebase` this to `maint`, you dragged in a lot of unrelated work when you merged `maint` into the branch.

File: lib/compiler/doc/notes.md [hunk 3]
```diff
@@ -2985,7 +2985,7 @@ This document describes the changes made to the Compiler application.
 
   Own Id: OTP-10706
 
-- Binary syntax matches could cause an internal consistency error in in the
+- Binary syntax matches could cause an internal consistency error in the
```
Comment (jhogberg) [inline lib/compiler/doc/notes.md:2988]: Release notes have already been published, there's no use correcting errors here.

**Review by jhogberg — COMMENTED** [review]
Thanks, there's just one more thing to fix that I missed before. Feel free to fix that and squash the commits :)

**Comment by jhogberg** [comment]
Please revert all the changes to `notes.md` and squash the PR down into one commit. :)

File: lib/compiler/doc/notes.md (hunk 3 above)
Comment (Ironjanowar) [reply lib/compiler/doc/notes.md:2988]: Sorry for all the inconvenience for such a dumb PR, I'm not used to rebasing and for the fix I text replaced all and I don't know the repository that well :sweat: 

Thanks for the patience :)

Comment (jhogberg) [reply lib/compiler/doc/notes.md]: It's not a dumb PR, see this as an exercise in learning `git rebase` :smile:

**Comment by jhogberg** [comment]
Merged, thanks again for the PR! :)

**Pull request merged**